### ISSUE TITLE
NSP check should pass

### ISSUE THREAD

**Issue by bbondy** [brave/brave-browser#130]
This is only used for the dev deps, but we should fix this:

```
$ nsp check
(+) 2 vulnerabilities found
┌────────────┬────────────────────────────────────────────────────────────────────┐
│            │ Prototype pollution attack                                         │
├────────────┼────────────────────────────────────────────────────────────────────┤
│ Name       │ hoek                                                               │
├────────────┼────────────────────────────────────────────────────────────────────┤
│ CVSS       │ 4 (Medium)                                                         │
├────────────┼────────────────────────────────────────────────────────────────────┤
│ Installed  │ 2.16.3                                                             │
├────────────┼────────────────────────────────────────────────────────────────────┤
│ Vulnerable │ <= 4.2.0 || >= 5.0.0 < 5.0.3                                       │
├────────────┼────────────────────────────────────────────────────────────────────┤
│ Patched    │ > 4.2.0 < 5.0.0 || >= 5.0.3                                        │
├────────────┼────────────────────────────────────────────────────────────────────┤
│ Path       │ brave@1.0.0 > less@2.7.3 > request@2.81.0 > hawk@3.1.3 >           │
│            │ hoek@2.16.3                                                        │
├────────────┼────────────────────────────────────────────────────────────────────┤
│ More Info  │ https://nodesecurity.io/advisories/566                             │
└────────────┴────────────────────────────────────────────────────────────────────┘

┌────────────┬────────────────────────────────────────────────────────────────────┐
│            │ Prototype pollution attack                                         │
├────────────┼────────────────────────────────────────────────────────────────────┤
│ Name       │ hoek                                                               │
├────────────┼────────────────────────────────────────────────────────────────────┤
│ CVSS       │ 4 (Medium)                                                         │
├────────────┼────────────────────────────────────────────────────────────────────┤
│ Installed  │ 2.16.3                                                             │
├────────────┼────────────────────────────────────────────────────────────────────┤
│ Vulnerable │ <= 4.2.0 || >= 5.0.0 < 5.0.3                                       │
├────────────┼────────────────────────────────────────────────────────────────────┤
│ Patched    │ > 4.2.0 < 5.0.0 || >= 5.0.3                                        │
├────────────┼────────────────────────────────────────────────────────────────────┤
│ Path       │ brave@1.0.0 > nsp@3.2.1 > wreck@12.5.1 > boom@5.2.0 > hoek@2.16.3  │
├────────────┼────────────────────────────────────────────────────────────────────┤
│ More Info  │ https://nodesecurity.io/advisories/566                             │
└────────────┴────────────────────────────────────────────────────────────────────┘
```

for the nsp chck, looks like it's included here:
```
Brians-MacBook-Pro:brave bbondy$ npm ls hoek
brave@1.0.0 /Users/bbondy/projects/brave/brave
├─┬ less@2.7.3
│ └─┬ request@2.81.0
│   └─┬ hawk@3.1.3
│     ├─┬ boom@2.10.1
│     │ └── hoek@2.16.3  deduped
│     ├── hoek@2.16.3
│     └─┬ sntp@1.0.9
│       └── hoek@2.16.3  deduped
└─┬ webpack@3.10.0
  └─┬ watchpack@1.4.0
    └─┬ chokidar@1.7.0
      └─┬ fsevents@1.1.3
        └─┬ node-pre-gyp@0.6.39
          └─┬ hawk@3.1.3
            ├─┬ boom@2.10.1
            │ └── hoek@2.16.3  deduped
            ├── hoek@2.16.3
            └─┬ sntp@1.0.9
              └── hoek@2.16.3  deduped
```

I tried updating all node modules with ncu, but it's not fixed.
I did a fresh npm install after clearing node_modules with that.

Maybe we should just add it to the ignored list?

By the way I did the same check in src/brave as well and it says `(+) No known vulnerabilities found`

**Comment by diracdeltas** [comment]
FWIW we would like to replace NSP with `npm audit`. may be a better use of resources to just focus on npm audit errors.

**Comment by jumde** [comment]
Fixed: c458814
